blog/_posts/AKS - Things you wish you knew!

```diff
@@ -0,0 +1,35 @@
+---
+title: "2024-07-24-AKS:Things you wish you knew!"
```

Comment:
You can remove the date from this title. This field represents how the post will be titled, do it should be something like "Things you wish you knew!"

The file name needs to have the date and follow an exact format. Such as: `2024-07-24-aks-things-you-wish-you-knew.md`